test/powershell/engine/ETS/WildcardPattern.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "WildCardPattern Tests" -Tag "CI" {
+    It "Should escape special characters from '<String>' to '<EscapedResult>' using Escape method" -TestCases @(
+        @{ String = 'abc`def'; EscapedResult = 'abc`def' }
+        @{ String = 'abc?def'; EscapedResult = 'abc`?def' }
+        @{ String = '?abcdef'; EscapedResult = '`?abcdef' }
+        @{ String = 'abc*def'; EscapedResult = 'abc`*def' }
+        @{ String = 'abc[def'; EscapedResult = 'abc`[def' }
+        @{ String = 'abc]def'; EscapedResult = 'abc`]def' }
+        @{ String = 'abcdef]'; EscapedResult = 'abcdef`]' }
+        @{ String = 'abc[]def'; EscapedResult = 'abc`[`]def' }
+    ) {
+        param($String, $EscapedResult)
+        $escaped = [WildcardPattern]::Escape($String)
+        $escaped | Should -BeExactly $EscapedResult
+
+        # -like comparison should also match escaped string from WildCardPattern.Escape
+        $String | Should -BeLike $escaped
```

Comment:
@iSazonov This was the test I was pointing out. This currently returns `False` without this change, and with the change it returns `True` so this is why I had it included.

Reply:
Here BeLike is Pester "function". My expectation is we _explicitly_ check `-like` (and its friends) operator. I think the explicit test should be in ComparisonOperator.Tests.ps1 to cover example from related issue that we really fix.

Reply:
Thanks @iSazonov. I think that's a better place. I have removed other tests and moved backtick specific ones to ComparisonOperator.Tests.ps1.

Please see 8aca5c8ef